data/obsidmerge_header_lookup.txt

```diff
@@ -54,3 +54,24 @@ TIMEUNIT   Fail;Default-s
 TIMESYS    Fail;Default-UTC
 TIMEPIXR   Fail;Default-0.5
 OBI_NUM    Skip
+THRFILE    Fail
```

Comment:
Having these as Fail created a bunch of messages like the following (once I fix some things). I think it would make sense for the obsidmerge case to have these as `Skip` rather than `Fail` - does that make sense?

```Combining 4 observations.
BIASFIL2 values are different...FAIL...
BIASFIL3 values are different...FAIL...
BIASFIL5 not present in all input files...FAIL...
BIASFIL6 values are different...FAIL...
...
```

Reply:
Yup, `Skip` is probably a better option.  I remember I wasn't sure if it was supported for string data-types (`Fail` wasn't until recently-ish) but a quick check looks like it is.